ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -426,6 +426,7 @@ tests:
       ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate-4-11
     env:
+      ALLOW_CONVERGED_FLOW: "True"
```

Comment:
If we do
```
    env:
      ASSISTED_ZTP_CONFIG: |
        ALLOW_CONVERGED_FLOW: "True"
```
then we don't need a new definition in `assisted-baremetal-operator-setup-ref.yaml`

Reply:
Forget, let's go your way

Reply:
The `ASSISTED_ZTP_CONFIG` belongs to the ztp-ref, we need the `ALLOW_CONVERGED_FLOW` in the operator-setup-ref.
Since this is currently the only one lets keep it simple